CodeStarterCamp_Week5/auditionManager.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  auditionManager.swift
+//  CodeStarterCamp_Week5
+//
+//  Created by MARY on 2023/03/20.
+//
+
+import Foundation
+
+struct AuditionManager {
+    let totalApplicantsList: [Person]
+    var passedApplicantsList: [Person] = []
+    
+    mutating func cast() {
+        guard !totalApplicantsList.isEmpty else {
+            print("지원자가 없습니다.")
+            return
+        }
+
+        for applicant in totalApplicantsList {
+            if let person = applicant as? BadPersonality, isBadPerson(person) { continue }
+
+            if let person = applicant as? Talent {
+                guard hasOutstandingTalent(person) else { continue }
+                passedApplicantsList.append(applicant)
+            }
```

Comment:
위에 작성해주신 것과 같은 방법으로 다음과 같이 쓰면 좀 더 깔끔하겠죠?
```suggestion
            guard let person = applicant as? Talent, hasOutstandingTalent(person) else { continue }
            passedApplicantsList.append(applicant)
```